app/javascript/onboarding/__tests__/Onboarding.test.jsx

```diff
@@ -189,16 +189,16 @@ describe('<Onboarding />', () => {
 
     test('should allow a user to add a tag and advance', async () => {
       fetch.once({});
-      const followTags = onboardingSlides.find(<FollowTags />);
-      const firstButton = onboardingSlides
-        .find('.onboarding-tags__button')
-        .first();
-
       expect(onboardingSlides.find('.next-button').text()).toContain(
         'Skip for now',
       );
 
+      const firstButton = onboardingSlides
+        .find('.onboarding-tags__button')
+        .first();
+
       firstButton.simulate('click');
+      const followTags = onboardingSlides.find(<FollowTags />);
```

Comment:
I had to move these declarations around to resolve `preact-render-spy: Performing operation on stale find() result` log warnings